### PR TITLE
firefox 53.3.0esr

### PR DESCRIPTION
**Pull request by protonesso**
out fresh esr release!

### CODE REVIEW

File: pkg/firefox [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-http://releases.mozilla.org/pub/mozilla.org/firefox/releases/52.2.1esr/source/firefox-52.2.1esr.source.tar.xz
-ftp://ftp.mozilla.org/pub/mozilla.org/firefox/releases/52.2.1esr/source/firefox-52.2.1esr.source.tar.xz
-
-[vars]
-filesize=212033732
-sha512=1d79e6e4625cf7994f6d6bbdf227e483fc407bcdb20e0296ea604969e701f551b5df32f578d4669cf654b65927328c8eb0f717c7a12399bf1b02a6ac7a0cd1d3
-pkgver=7
-tardir=firefox-52.2.1esr
-uchkurl=http://releases.mozilla.org/pub/mozilla.org/firefox/releases/
-no_unlink=1
```
Comment (Francesco149) [inline pkg/firefox]: please test your stuff before making a pull request

you removed filesize, hash and tardir
this probably doesn't compile, or if it does there will be no integrity check

**Comment by rofl0r** [comment]
heh, the issue number matches the firefox version...

**Comment by protonesso** [comment]
fix

**Comment by rofl0r** [comment]
sorry d00d, but we don't merge PRs that have not even been tested

**Comment by rofl0r** [comment]
version that actually works: https://github.com/sabotage-linux/sabotage/commit/00e55d47acaef1690b624e90e225a4831f42a42a

**Pull request closed without merging**